app/routes.php

```diff
@@ -39,6 +41,13 @@
 
         $versionGroup->post('/auth', Login::class)
             ->add(CredentialsRecaptchaMiddleware::class); // Runs last, after group's IP + rate limit middlewares.
+
+        $versionGroup->post(
+            '/password-reset-token',
+            CreatePasswordResetToken::class
+        ); // @todo probably should put  recapcha on this. Also needs rate limiting.
```

Comment:
NoelLH can you advise about rate limiting and captcha? Thanks.

Reply:
@bdsl I would definitely agree we should have both to avoid brute forcing.

I don't think the levels or tokens need to vary compared to other use cases, so it should just be a case of copying the same middleware that's on other routes + groups.

I think that to make it work behind a load balancer and get the behaviour described on the above endpoint, if chaining with a fluent interface then the best order to list them is:
1. recaptcha (needs the IP workaround)
2. IP
3. rate limit (understands the forwarded for header on its own)

Reply:
Thanks, I think I've added the necessary middlewares.  The code in https://github.com/thebiggive/donate-frontend/pull/815/ will need adjusted to send the `x-captcha-code` custom header.